clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -277,12 +285,22 @@ export default async function handler(
     `
       : ""
   }${
-    e2eQuery === "true"
+    e2eQuery === "true" // let Cypress set the config and initialize fides.js
       ? ""
       : `
-        // Initialize fides.js with custom config
     var fidesConfig = ${fidesConfigJSON};
-    window.Fides.init(fidesConfig);`
+    ${
+      initializeQuery === "false"
+        ? `// Capture the base config for later initialization
```

Comment:
I think this would be easier to understand if you flipped this around a bit and did the following:
1. Always set `window.Fides.config`
2. Call `window.Fides.init` unless initialize query is false

It's a single difference, but as is the Fides.config variable is only set in one branch, which feels like a foot gun to me